ws-nextjs-app/cypress/e2e/languagesPage/tests.ts

```diff
@@ -0,0 +1,120 @@
+import localData from '#data/pidgin/homePage/index.json';
+
+export const homePageTestSuites = [
+  {
+    path: '/ws/languages',
+    service: 'pidgin',
+    runforEnv: ['local', 'test'],
+    tests: [
+      ({ service, pageType }: { service: string; pageType: string }) => {
+        describe(`Tests for ${pageType} page - ${service}`, () => {
+          const currentEnv = Cypress.env('APP_ENV');
+
+          if (currentEnv === 'local') {
+            it('should show Local Pidgin fixture data when no renderer_env is set', () => {
+              cy.get('nav').should('exist');
+              cy.get('main').should('exist');
+
+              cy.get('main').within(() => {
+                cy.get('h1').should('contain', localData.data.title);
+                cy.get('meta[name="description"]').should(
+                  'have.attr',
+                  'content',
+                  localData.data.description,
+                );
+              });
+            });
+          } else {
+            it('should show Test Pidgin Homepage when no renderer_env is set', () => {
+              cy.request({
+                url: 'https://www.test.bbc.com/pidgin',
+                headers: {
+                  Origin: 'https://www.test.bbc.com',
+                },
+              }).then(response => {
+                const testContent = response.body;
+
+                cy.get('nav').should('exist');
+                cy.get('main').should('exist');
+
+                cy.get('main').then($main => {
+                  const localContent = $main.html();
+                  expect(localContent).to.include(testContent); // TODO: improved comparison
+                });
+              });
+            });
+          }
+
+          it('should show Test Pidgin Homepage when renderer_env=test', () => {
+            cy.request(
+              `${Cypress.config().baseUrl}/ws/languages?renderer_env=test`,
```

Comment:
We can't use renderer_env in local E2E tests, as we don't have access to the BFF. These tests will only work on test/live